### PR TITLE
Added documentation for ansible installer modifications for the default backends and fileshare parameters

### PR DESCRIPTION
**Pull request by bhagyashree-sarawate**
Hi Team,
Added the documentation for two additional parameters for the file share feature 
Also added the installer behaviour for the default backends 
Please review
Regards
Bhagyashree Sarawate

### CODE REVIEW

File: ansible_3par_docker_plugin/README.md [hunk 1]
```diff
@@ -54,7 +54,27 @@ These playbooks perform the following tasks on the Master/Worker nodes as define
...
+      |BLOCK   |-- |FILE |Plugin successfully installs.|
+      |FILE    |BLOCK |-- |Plugin successfully installs.|
+      |BLOCK   |BLOCK |FILE |When we have DEFAULT backend with Block driver, then there should not be any DEFAULT_BLOCK backend in multibackend configuration.Plugin installation fails in this case.|
+      |FILE    |BLOCK |FILE |When we have DEFAULT backend with File driver, then there should not be any DEFAULT_FILE backend in multibackend configuration.Plugin installation fails in this case.|
+      |BLOCK   |FILE |--  |DEFAULT_BLOCK is not allowed to be configured for File driver. Plugin installation fails in this case.|
+      |FILE   |-- |BLOCK  |DEFAULT_FILE is not allowed to be configured for Block driver. Plugin installation fails in this case.|
+      |BLOCK   |BLOCK |-- |When we have DEFAULT backend with Block driver, then there should not be any DEFAULT_BLOCK backend in single backend configuration.Plugin installation fails in this case.|
+      |FILE    |-- |FILE |When we have DEFAULT backend with File driver, then there should not be any DEFAULT_FILE backend in single backend configuration.Plugin installation fails in this case.|
+
```
Comment (sneharai4) [inline ansible_3par_docker_plugin/README.md]: @bhagyashree-sarawate Can you please add what BLOCK and FILE means here as a Note.

Example:
FILE points to hpedockerplugin_driver: hpedockerplugin.hpe.hpe_3par_file.HPE3PARFileDriver when specified in plugin_configuration_properties.yml.
Similarly specify for BLOCK.

Also, we can maintain a template for the "INSTALLER BEHAVIOR". E.g. Plugin installation successful.
Plugin installation fails. <REASON>

Comment (bhagyashree-sarawate) [reply ansible_3par_docker_plugin/README.md]: i have added the description about the FILE and BLOCK.
Do we have INSTALLER_BEHAVIOR and comments as an another  section, which describes the reason for failure as i have mentioned

**Comment by sneharai4** [comment]
@bhagyashree-sarawate Please resolve the conflicts.

**Comment by bhagyashree-sarawate** [comment]
> @bhagyashree-sarawate Please resolve the conflicts.

@sneharai4 done, please verify

**Pull request merged**